cmd/trade.go

```diff
@@ -500,6 +505,21 @@ func runTradeCmd(options inputs) {
 	botConfig = convertDeprecatedBotConfigValues(l, botConfig)
 	l.Infof("Trading %s:%s for %s:%s\n", botConfig.AssetCodeA, botConfig.IssuerA, botConfig.AssetCodeB, botConfig.IssuerB)
 
+	tracker := metrics.MakeMetricsTracker(
+		version,
+		runtime.GOOS,
+		*options.ui,
+		*options.strategy,
+		float64(botConfig.TickIntervalSeconds),
+		botConfig.TradingExchange,
+		fmt.Sprintf("%s:%s/%s:%s", botConfig.AssetCodeA, botConfig.IssuerA, botConfig.AssetCodeB, botConfig.IssuerB),
```

Comment:
this tradingPair is not correct. if `botConfig.IsTradingSdex()` then this is correct, else it's just `codeA/codeB` (i.e. no issuers).

not sure if this logic is specified elsewhere in the codebase (probably is) but if you can't find it then it's ok to put this in a separate function on `botConfig` struct so we can reuse it later